samples/get_sample_data.bash

```diff
@@ -0,0 +1,18 @@
+#! /usr/bin/env bash
+
+set -e
+
+# save a few directory names
+CWD=$PWD
+SAMPLES="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+git submodule init
+
+# Get the sample data
+cd $SAMPLES/VIC_sample_data
```

Comment:
yes. This is the empty submodule that we're going to fill with the next commands.